<?xml version="1.0" encoding="utf-8"?>
<testsuites tests="11" failures="0" disabled="0" errors="5" time="11" name="">
	<testsuite name="addTest" tests="0" failures="" disabled="" errors="0" time="">
		<testcase name="case 1" status="run" time="" classname="addTest" />
	</testsuite>
	<testsuite name="SetDtatTest" tests="0" failures="" disabled="" errors="0" time="">
		<testcase name="case 1" status="run" time="" classname="SetDtatTest" />
	</testsuite>
	<testsuite name="SetGlobalTestTest" tests="0" failures="" disabled="" errors="0" time="">
		<testcase name="case 1" status="run" time="" classname="SetGlobalTestTest" />
	</testsuite>
	<testsuite name="CallScriptTestTest" tests="1" failures="" disabled="" errors="1" time="">
		<testcase name="case 1" status="run" time="" classname="CallScriptTestTest">
			<failure message="case 1: signal SIGSEGV: Invalid memory reference, stop." type="error">&lt;!case 1: signal SIGSEGV: Invalid memory reference, stop.&gt;</failure>
		</testcase>
	</testsuite>
	<testsuite name="localIOTest" tests="0" failures="" disabled="" errors="0" time="">
		<testcase name="case 1" status="run" time="" classname="localIOTest" />
	</testsuite>
	<testsuite name="CallScriptTest_3Test" tests="0" failures="" disabled="" errors="0" time="">
		<testcase name="case 1" status="run" time="" classname="CallScriptTest_3Test" />
	</testsuite>
	<testsuite name="FunctionPtrStruct_3Test" tests="0" failures="" disabled="" errors="0" time="" />
	<testsuite name="FunctionPtrTest_3Test" tests="0" failures="" disabled="" errors="0" time="">
		<testcase name="case 1" status="run" time="" classname="FunctionPtrTest_3Test" />
	</testsuite>
	<testsuite name="GlobalTest_3Test" tests="4" failures="" disabled="" errors="0" time="">
		<testcase name="case 1" status="run" time="" classname="GlobalTest_3Test" />
		<testcase name="case 2" status="run" time="" classname="GlobalTest_3Test" />
		<testcase name="case 3" status="run" time="" classname="GlobalTest_3Test" />
		<testcase name="case 4" status="run" time="" classname="GlobalTest_3Test" />
	</testsuite>
	<testsuite name="localIO_3Test" tests="0" failures="" disabled="" errors="0" time="">
		<testcase name="case 1" status="run" time="" classname="localIO_3Test" />
	</testsuite>
	<testsuite name="MockTest_3Test" tests="0" failures="" disabled="" errors="0" time="">
		<testcase name="case 1" status="run" time="" classname="MockTest_3Test" />
	</testsuite>
	<testsuite name="pow_3Test" tests="4" failures="" disabled="" errors="2" time="">
		<testcase name="case 1" status="run" time="" classname="pow_3Test" />
		<testcase name="case 2" status="run" time="" classname="pow_3Test" />
		<testcase name="case 3" status="run" time="" classname="pow_3Test">
			<failure message="case 3: return expected 3 but 4" type="error">&lt;!case 3: return expected 3 but 4&gt;</failure>
		</testcase>
		<testcase name="case 4" status="run" time="" classname="pow_3Test">
			<failure message="case 4: return expected 4 but 27" type="error">&lt;!case 4: return expected 4 but 27&gt;</failure>
		</testcase>
	</testsuite>
	<testsuite name="localIO_2Test" tests="2" failures="" disabled="" errors="2" time="">
		<testcase name="case 1" status="run" time="" classname="localIO_2Test">
			<failure message="case 1: return expected 0 but 102400" type="error">&lt;!case 1: return expected 0 but 102400&gt;</failure>
		</testcase>
		<testcase name="case 2" status="run" time="" classname="localIO_2Test">
			<failure message="case 2: return expected 0 but 1" type="error">&lt;!case 2: return expected 0 but 1&gt;</failure>
		</testcase>
	</testsuite>
</testsuites>
